dev-docs/bidders/1accord.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: bidder
+title: 1accord
+description: Prebid 1accord Bidder Adapter
+pbjs: true
+biddercode: 1accord
+aliasCode : nexx360
+gvl_id: 965
```

Comment:
Since you're using Nexx360's GVL ID, you need to do two things:

- confirm that 1accord does not have access to detailed log information including IP address. If you do, then you need to get your own GVL ID.
- update this value to "965 (Nexx360)"

Reply:
We do confirm about log.
The value has been also updated.